spec/factories/trainees.rb

```diff
@@ -3,8 +3,14 @@
 # rubocop:disable Style/SymbolProc
 FactoryBot.define do
   factory :abstract_trainee, class: Trainee do
-    sequence :trainee_id do |n|
-      n.to_s
+    sequence :trainee_id do |n, thing|
+      year = if commencement_date
```

Comment:
What should the YY/YY relate to? `commencement_date`, `course_start_date`, `course_end_date`?

Reply:
@FriedSock probably the course start date as we'll soon be basing the academic year on this - and this field is similar.

So for courses starting in September 2021, their academic year is 2021-2022.